desktop/src/main/java/bisq/desktop/main/content/academy/overview/OverviewAcademyView.java

```diff
@@ -112,6 +113,7 @@ private void addSmallBox(String leftIconId,
                 Res.get("academy.overview." + leftTopic),
                 "bisq-text-headline-2",
                 leftIconId,
+                5d,
```

Comment:
Actually the problem is inside the icon images. Some have a free space to the right thus increasing the gap. Better leave it to 10px as 5px is too small for those images without gap. The images need to get updated, but that can be done by https://github.com/axpoems when he is cleaning up all images.

Reply:
done